static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -118,7 +118,15 @@ define([
 
     function initPlayButtons(root) {
         fastdom.read(function () {
-            $('.js-video-play-button', root).each(function (el) {
+            $('.js-video-play-button', root).each(function (el, options, duration) {
+                function initButtonDuration() {
```

Comment:
I would declare this function outside of the `.each` so that its declared once in total, rather than once per loop